examples/grpc/yclient/main.go

```diff
@@ -0,0 +1,62 @@
+package main
+
+import (
+	"fmt"
+	"log"
+	"time"
+
+	"go.uber.org/yarpc"
+	"go.uber.org/yarpc/encoding/raw"
+	"go.uber.org/yarpc/transport"
+	"go.uber.org/yarpc/transport/grpc"
+
+	"golang.org/x/net/context"
+)
+
+func main() {
+	dispatcher := yarpc.NewDispatcher(yarpc.Config{
+		Name: "hello",
+		Outbounds: transport.Outbounds{
+			"yarpc": grpc.NewOutbound("localhost:50054"),
```

Comment:
Does GRPC accept requests to all paths of the URL? If not, this should be a
complete URL like with the HTTP outbound.